.gitignore

```diff
@@ -3,7 +3,7 @@ photon_data/
 dependency-reduced-pom.xml
 .gradle/
 build
-
+.DS_Store
```

Comment:
I'd prefer if ignores specific to the local setup (MacOS here) go into the `.gitignore` in your home directory. This file should only have the ones specific to the project. (And, yes, the file needs some cleanup but that's for an extra PR.)